components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1140,6 +1147,8 @@ interface BraveWalletService {
 
   IsBase58EncodedSolanaPubkey(string key) => (bool result);
 
+  Base58Encode(array<array<uint8>> addresses) => (array<string> addresses);
```

Comment:
Just for my own context around this message processing model, what's the value in passing these over the IPC to encode them rather than handle them within the same process? Encoding seems like a fairly trivial thing to implement so just trying to understand this pattern better.

Reply:
Check out this discussion on the topic https://github.com/brave/brave-core/pull/13734#discussion_r894749025.  Basically, we could do this on the js side, but don't want to add another dependency.

Reply:
Awesome, thanks that makes sense to me in terms of the reasoning for this tradeoff